src/pages/projects/the-beginner-s-guide-to-vue-3/index.tsx

```diff
@@ -0,0 +1,230 @@
+            </div>
+            <p className="text-lg md:text-2xl leading-6 text-gray-500">
+              Portfolio Project
+            </p>
+            <h1 className="text-2xl md:text-4xl font-bold mt-2">
+              {course.projects.title}
+            </h1>
+          </div>
+          <div className="flex flex-col justify-center items-start mx-auto max-w-screen-md mb-16 w-full px-4">
+            <main className="prose dark:prose-dark prose-lg max-w-none w-full">
+              <Markdown>{introduction.text}</Markdown>
+              <Markdown>{projectBrief.text}</Markdown>
+              <section className="grid grid-cols-2 gap-4">
+                <Markdown className="col-span-1">
+                  {appRequirements.text}
+                </Markdown>
+                <Markdown className="rounded mt-20">{appData.text}</Markdown>
+              </section>
+              <Markdown>{developmentStandards.text}</Markdown>
+
+              <ProjectCTA
+                title="Show you're hard won Vue skills. Make it stick."
+                meta="Fork the codesandbox and get started with your solution."
+                buttonText="Start the Vue Fundamentals Project"
+                projectLink={course.projects.projectLink}
+              />
+              <Markdown>{stretchGoal.text}</Markdown>
+              <Markdown>{appDesign.text}</Markdown>
+              <Markdown>{productPage.description}</Markdown>
+              <Image
+                src={productPage.url}
+                height="826"
+                width="736"
+                className="rounded-md z-0"
+              />
+              <section className="grid grid-cols-2 mt-4 gap-4">
+                <Markdown className="col-span-1">
+                  {productCard.description}
+                </Markdown>
+                <Image
+                  src={productCard.url}
+                  height="471"
+                  width="296"
+                  className="rounded-md z-0"
+                />
+              </section>
+              <Markdown>{Submission.text}</Markdown>
+              <a
+                className="inline-flex justify-center items-center px-6 py-4 font-semibold rounded-md bg-blue-600 text-white transition-all hover:scale-105 transform hover:shadow-xl  hover:bg-blue-700 ease-in-out duration-200"
+                title="Share on twitter"
+                onClick={() =>
+                  track(`clicked Vue 3 fundamentals challenge CTA`)
+                }
+                href={course.projects.tweetCTA}
+                rel="noopener"
+                style={{
+                  color: 'white',
+                  textDecoration: 'none',
+                }}
+              >
+                Tweet @eggheadio
+              </a>
+            </main>
+          </div>
+        </div>
+      </div>
+    </>
+  )
+}
+
+type projectCTAProps = {
+  title: string
+  meta: string
+  buttonText: string
+  projectLink: string
+}
+
+const ProjectCTA = ({
```

Comment:
Wondering about supplying the data for this ProjectCTA from sanity, currently I have it hardcoded above